### PR TITLE
GeoDCAT support

### PR DESCRIPTION
**Pull request by kalxas**

### CODE REVIEW

File: ckanext/publicamundi/lib/actions/package.py [hunk 1]
```diff
@@ -60,6 +61,40 @@ def dataset_export(context, data_dict):
...
+    pkg = _get_action('package_show')(context, data_dict)
+    dtype = pkg.get('dataset_type')
+    obj = pkg.get(dtype) if dtype else None
+    result = None
+    if obj:
+        # Get a proper serializer
+        xser = xml_serializer_for(obj)
+        xser.target_namespace = config.get('ckan.site_url')
+        # Get the XML
+        tmp_xml = xser.dumps()
```
Comment (drmalex07) [inline ckanext/publicamundi/lib/actions/package.py]: Probably these 2 steps of serializing (`dumps`) and parsing (`fromstring`)  are redundant. The `IXmlSerializer` interface can return an etree object directly (see https://github.com/PublicaMundi/ckanext-publicamundi/blob/master/ckanext/publicamundi/lib/metadata/ibase.py#L53)

Comment (kalxas) [inline ckanext/publicamundi/lib/actions/package.py]: thanks, will fix

File: ckanext/publicamundi/plugins.py [hunk 2]
```diff
@@ -159,6 +159,10 @@ def before_map(self, mapper):
             m.connect(
                 '/api/publicamundi/dataset/export/{name_or_id}', 
                 action='dataset_export')
+
+            m.connect(
+                '/api/publicamundi/dataset/dcat/{name_or_id}',
```
Comment (drmalex07) [inline ckanext/publicamundi/plugins.py]: I suggest to name this as `dataset/export_dcat`, as this is also an export action.

File: ckanext/publicamundi/plugins.py [hunk 3]
```diff
@@ -236,6 +240,7 @@ def get_actions(self):
             'mimetype_autocomplete': ext_actions.autocomplete.mimetype_autocomplete,
             'dataset_export': ext_actions.package.dataset_export,
             'dataset_import': ext_actions.package.dataset_import,
+            'dcat_export': ext_actions.package.dcat_export,
```
Comment (drmalex07) [inline ckanext/publicamundi/plugins.py]: As before, i suggest `ext_actions.package.dataset_export_dcat`

File: ckanext/publicamundi/plugins.py (hunk 2 above)
Comment (kalxas) [inline ckanext/publicamundi/plugins.py]: Then the original one should be renamed export_iso or export_inspire

Comment (drmalex07) [inline ckanext/publicamundi/plugins.py]: Well, i considered the original one as the default exporting action (as ISO is a meaningful name only for INSPIRE schema, not for other arbitrary schemata).

Comment (drmalex07) [inline ckanext/publicamundi/plugins.py]: Also, assuming that GeoDCAT transformation is valid only on ISO-derived schemata, maybe we should throw an exception if one tries to apply it on an unrelated schema.

File: ckanext/publicamundi/lib/actions/package.py (hunk 1 above)
Comment (kalxas) [inline ckanext/publicamundi/lib/actions/package.py]: Fixed in https://github.com/kalxas/ckanext-publicamundi/commit/ceb196178a05aad4348565214c55fb5dd7b91336

File: ckanext/publicamundi/plugins.py (hunk 2 above)
Comment (kalxas) [inline ckanext/publicamundi/plugins.py]: Fixed

**Pull request merged**